doc/user/content/guides/node-js.md

```diff
@@ -132,6 +124,8 @@ client.connect((err, client) => {
 
 Querying Materialize is identical to querying a traditional PostgreSQL database: Node.js executes the query, and Materialize returns the state of the view, source, or table at that point in time.
 
+Because Materialize maintains materialized views in memory, response times are much faster than traditional database queries and polling (repeatedly querying) a view doesn't impact performance.
```

Comment:
It's not required, but putting a comma after "queries" might make the sentence more readable.